src/main/java/wolox/gmjavatraining/models/Book.java

```diff
@@ -0,0 +1,54 @@
+package wolox.gmjavatraining.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Getter @Setter
+    @Column
```

Comment:
@Column not necessary unless you need to specify some attribute for it, like it's name.
And if at some point you don't want to persist a column, just add the `transient` keyword to the variable declaration.